packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -110,6 +114,12 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
    */
   @Prop() collapsible?: boolean;
 
+  /**
+   * The specified height in rem to allow the answer to show before collapsing.
```

Comment:
```suggestion
   * The maximum height (in rem units) of the component when collapsed. 
```

though again I'm not sure this should be a "maximum".

Reply:
I am changing the word component to answer, we are talking about the answer height not the whole component of CRGA